cellmlmanip/model.py

```diff
@@ -1064,26 +1067,26 @@ def _convert_variable_instance(self, original_variable, cf, units, direction):
         return new_variable
 
 
-class NumberDummy(sympy.Dummy):
+class Quantity(sympy.Dummy):
     """
     Used to represent a number with a unit, inside a Sympy expression.
 
     Unlike sympy expressions, this number type will never be removed in simplify operations etc.
 
-    Number dummies should never be created directly, but always via :meth:`Model.add_number()`.
+    Quantities should never be created directly, but always via :meth:`Model.create_quantity()`.
 
     Assumes the value is real.
 
     To get the actual value as a float or string, use ``float(dummy)`` or ``str(dummy)`` respectively.
-    You can also use ``dummy.evalf()`` to get the value as a :class:`sympy.Float`.
+    You can also use ``quantity.evalf()`` to get the value as a :class:`sympy.Float`.
     """
 
     # Sympy annoyingly overwrites __new__
     def __new__(cls, value, *args, **kwargs):
         # Middle argument is the symbol name, so must be a string
         if not isinstance(value, str):
             value = '{:g}'.format(value)
-        return super().__new__(cls, value, real=True)
+        return super().__new__(cls, '_' + value, real=True)
```

Comment:
I have no idea why sympy stopped adding the underscore itself when I renamed the class, but apparently it did!

Reply:
that seems off you sure it's not a different version of sympy?

Reply:
I don't think so - I didn't change any library versions locally, only change was this PR itself.